internal/sync/sync.go

```diff
@@ -67,6 +67,37 @@ func (s Sync) syncPaths(delete bool, local, remote string) error {
 	return nil
 }
 
+func (s Sync) remoteMkDir() error {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
+	defer cancel()
+
+	conn, err := s.Client.DialWsep(ctx, s.Environment)
+	if err != nil {
+		return err
+	}
+	defer conn.Close(websocket.CloseNormalClosure, "")
+
+	execer := wsep.RemoteExecer(conn)
+	process, err := execer.Start(ctx, wsep.Command{
+		Command: "mkdir",
+		Args:    []string{"-p", s.RemoteDir},
+	})
+	if err != nil {
+		return err
+	}
+	go io.Copy(os.Stdout, process.Stderr())
+	go io.Copy(os.Stderr, process.Stdout())
+
+	err = process.Wait()
+	if code, ok := err.(wsep.ExitError); ok {
+		return fmt.Errorf("mkdir exit status: %v", code)
+	}
+	if err != nil {
+		return xerrors.Errorf("execution failure: %w", err)
+	}
+	return nil
```

Comment:
Would it make sense to move the parts that's shared with `remoteRm` to its own function? Maybe something like `remoteCommand(ctx context.Context, command string, args []string) error`.

Reply:
Seems like a good candidate to refactor. I'll look at it